src/asana/helpers.py

```diff
@@ -75,7 +75,9 @@ def default_due_date_str(reference_datetime: datetime = None) -> str:
 
 
 def _task_status_from_pull_request(pull_request: PullRequest) -> str:
-    if not pull_request.closed():
+    if not pull_request.closed() and pull_request.is_draft():
+        return "Draft"
+    elif not pull_request.closed() and not pull_request.is_draft():
         return "Open"
     elif pull_request.closed() and pull_request.merged():
         return "Merged"
```

Comment:
i.e.
```py
elif pull_request.closed():
    return "Merged" if pull_request.merged() else "Closed"
```

I think we can actually just make that an `else` and drop the logger error as I believe it is unreachable.

Reply:
Cool yeah either way works for me - consistency is great, so either way you pick, can you make sure it's consistent between open/closed logic?